### ISSUE TITLE
Constellations need 2D and 3D sky checked to turn on in Sky mode

### ISSUE THREAD

**Issue by astrodavid10** [WorldWideTelescope/wwt-web-client#335]
In order for constellation lines/figures/labels/boundaries to appear while in sky mode, each of the following need to be checked "on" in the layer manager: 

"3D Universe"  
"2D Sky"
"Show Solar System"

This is strange but needs fixing because there are times when you want to view Sky mode with constellations but no planets / sun.

*don't ask how long it took to unravel this as the reason that my tour was broken

**Comment by pkgw** [comment]
Yeah, it seems like you shoudn't need to have all of these checked. Web? Windows? Both?

**Comment by astrodavid10** [comment]
Web only, Windows works
